### PR TITLE
Merge input comp.

### PR DESCRIPTION
**Pull request by piyushsh25**
image

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    padding: var(--size12);
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+label {
+    margin: var(--sizee3) var(--size0) var(--size0) var(--size0);
+    font-size: var(--size9);
+}
+
```
Comment (lannisterrr) [inline components/input/input.css:14]: Looks fine to me.

**Review by abhij1607 — APPROVED** [review]
looks good

**Review by FarhanMobashir — COMMENTED** [review]
✅✅

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="input.css">
+    <title>Input</title>
```
Comment (MaruthiPothuganti) [inline components/input/input.html:9]: Suggestion : better to have a title like EnginUI | Input

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended change on title of page

**Pull request merged**